### DISCUSSION TITLE
FileExportCall with `gid` param

### DISCUSSION THREAD

**Discussion by oknozor** [Byron/google-apis-rs#360] [Q&A] [answered]
Hello, I am trying to export some spreadsheet to csv, so far it works fine : 
```rust
    let mut response = hub
        .files()
        .export("1XIgh2LH_9d6Ov8ux-P2EZrMItwNHRNQYAs-1rkCFVY0", "text/csv")
        .doit()
        .await?;
 ```

But some of the spreadsheets have multiple tabs, I need to export each tab separately so I tried the following: 
```rust
    let mut response = hub
        .files()
        .export("1XIgh2LH_9d6Ov8ux-P2EZrMItwNHRNQYAs-1rkCFVY0", "text/csv")
        .param("gid", "12345")
        .doit()
        .await?;
 ```
 
 This doesn't seem to work as expected. Maybe I am missing something or this is not supported yet ?

**Comment by Byron** [comment, marked as answer]
The answer you seek should be available in the official google documentation for the API, and in theory that is applicable to the generated crate.

There cannot be Google API specific support here, only support for the generator which produces crates from Google API definitions.

**Reply by oknozor** [reply]
Thanks, I managed to do what I wanted using another route.
